src/Microsoft.Health.Fhir.Anonymizer.Stu3.CommandLineTool/configuration-sample.json

```diff
@@ -62,7 +62,7 @@
     {"path": "FamilyMemberHistory.deceased", "method": "redact"},
     {"path": "FamilyMemberHistory.condition.onset as Age", "method": "perturb", "span": 0.1, "rangeType": "proportional", "roundTo": 0},
     {"path": "FamilyMemberHistory.condition.onset", "method": "redact"},
-    {"path": "ClinicalImpression.description", "method": "redact"},  
+    {"path": "ClinicalImpression.description", "method": "presidio"},  
```

Comment:
Ok, for now.
I think in the future we might want to generalize this.
for example: "method": "unstructured-data-replace"
etc, and use a configuration to imply which "unstructured-data" anonymizer to use.

Reply:
makes sense